src/CONST.ts

```diff
@@ -2376,6 +2376,7 @@ const CONST = {
             TRACK: 'track',
         },
         AMOUNT_MAX_LENGTH: 8,
+        DISTANCE_REQUEST_AMOUNT_MAX_LENGTH: 10,
```

Comment:
```suggestion
        DISTANCE_REQUEST_AMOUNT_MAX_LENGTH: 14,
```

Backend allows for any number of digits and this seems to be the maximum possible amount for distance expense.